.gitlab-ci.yml

```diff
@@ -502,6 +505,10 @@ SonarQube:
 
 finish:
   stage: finish
+  rules: 
```

Comment:
is this really needed? Shouldn't the finish job just run always?

Reply:
I need to check it, I remember that it happened to me once that everything was skipped and only init and finish ran which reported pass to the PR.

Reply:
Yeah it's not necessary, I once removed the `SKIP_CI` variable from Gitlab and it didn't run any tests, just init and finish which reported pass on the PR. This should not happen though if we keep the variable in CI.. It's currently set to `false`.